app/src/main/java/org/simple/clinic/widgets/PatientSearchResultItemView.kt

```diff
@@ -77,10 +92,17 @@ class PatientSearchResultItemView(
     }
   }
 
-  private fun renderPatientPhoneNumber(phoneNumber: String?) {
-    phoneNumberContainer.visibleOrGone(phoneNumber.isNullOrBlank().not())
-    if (phoneNumber != null) {
-      phoneNumberTextView.text = phoneNumber
+  private fun renderPatientPhoneNumber(number: PhoneNumber) {
+    when (number) {
+      is PhoneNumber.Highlighted -> {
+        val highlightNumber = SpannableStringBuilder(number.patientNumber)
+        val highlightColor = ContextCompat.getColor(context, R.color.blue2)
```

Comment:
This color will no longer exist, can we create a new color for representing the instant search highlight, or you can use `simple_light_blue_100`